services/services.rst

```diff
@@ -0,0 +1,209 @@
+Overview of services
+******
+
+Below are tools and resources that are used in the PFS project. 
+Many services are hosted at IPMU under `pfs.ipmu.jp <https://pfs.ipmu.jp>`_ 
+(mainly for instrument hardware development) or 
+`pfspipe.ipmu.jp <pfspipe.ipmu.jp>`_ (for software development), and also PFS 
+project is using several external services, like 
+`GitHub Subaru-PFS organization <https://github.com/Subaru-PFS>`_. 
+You may need account(s) and access to some or all of them depending on your 
+roles and contributions to the project, most of services hosted at IPMU are 
+under single sign on (SSO) by pfs.ipmu.jp. 
+Refer `account policy <account.rst>`_ for details and registration. 
+
+* `SuMIRe/PFS PBworks wiki`_
+* `Public Web Site at pfs.ipmu.jp`_
+* `PFS project internal website (pfs.ipmu.jp)`_
+* `Online meeting`_
+* `Services for software development at pfspipe.ipmu.jp and external`_
+
+SuMIRe/PFS PBworks wiki
+------
+
+"SuMIRe/PFS PBworks" is the only official wiki system for PFS. 
+
+======
+Login and ToC
+======
+
+After logging in at `PBworks <http://sumire.pbworks.com/>`_, 
+you will get a project index page named 
+"Subaru Measurement of Images and Redshifts (SuMIRe)". 
+This page contains links to contents in the wiki, such as 
+
+* PFS Project Office (General links, including link to telecon indexes)
+* Documents (Link list to documents)
+* Meeting, conference, etc.  (Link list to meetings)
+* Mailing lists (List of avail lists)
+* PFS working groups (Member list)
+
+======
+Page indexes
+======
+
+For some continuous meetings, index for each agenda/memo pages are avail.
+
+* `Systems Engineering group telecon <http://sumire.pbworks.com/Systems-engineering-telecons>`_
+* `Manager group telecon <http://sumire.pbworks.com/Management-telecons>`_
+
+
+======
+Files
+======
+
+Also, you can find all files uploaded into this SuMIRe/PFS PBworks from 
+`folder listing <http://sumire.pbworks.com/w/browse/#view=ViewAllFiles>`_.
+Some files are categorized into *FOLDERS*, and you can get each list by 
+clicking FOLDER name at left side. 
+
+======
+Editing manual
+======
+
+You can find `manual for editing PBworks <http://usermanual.pbworks.com/>`_. 
+
+
+Public Web Site at pfs.ipmu.jp
+------
+
+Public project information is at http://pfs.ipmu.jp , 
+such as list of meetings, list of publications, and instrument parameters. 
+
+PFS project internal website (pfs.ipmu.jp)
+------
+
+Every contents at https://pfs.ipmu.jp/ are project only, and you will 
+be required to log in to view pages. 
+
+If you have any issue on this site, contact administrator 
+at pfs at pfs.ipmu.jp . 
```

Comment:
contact pfs at pfs.ipmu.jp  ?

Reply:
seems better to refer account.rst#lost-account-or-change-password